model/Xsc/Event/StartScanRequest.ts

```diff
@@ -0,0 +1,15 @@
+import { ScanEventType, ScanEventStatus } from './index';
+
+export interface StartScanRequest {
+    event_type?: ScanEventType;
+    event_status?: ScanEventStatus;
+    product?: string;
```

Comment:
I suggest product to be a closed enum (maybe add "other" to support)

Reply:
IMO it's ok to be an open list, there is no need to constraint this.